src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,109 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private int capacity = 10;
```

Comment:
it would be better if `capacity` was a constant (`private static int INITIAL_CAPACITY`)
and after that we can use `elementData.length` to get info about current capacity